src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -8,21 +8,29 @@
 import * as path from 'path'
 import * as vscode from 'vscode'
 
-import { DebugConfiguration } from '../../lambda/local/debugConfiguration'
+import { PythonDebugConfiguration } from '../../lambda/local/debugConfiguration'
+import { writeFile } from '../filesystem'
+import { fileExists, readFileAsString } from '../filesystemUtilities'
 import { LambdaHandlerCandidate } from '../lambdaHandlerSearch'
 import { getLogger } from '../logger'
```

Comment:
Ditch this in favor of `getChannelLogger.logger`

Reply:
Sometimes I just want to do debug style logging and not to an official pretty channel with associated NSL message key and template. For these getLogger() fits.

Reply:
No changes needed.